### ISSUE TITLE
Send test email is not working

### ISSUE THREAD

**Issue by lasat** [globaleaks/GlobaLeaks#3259]
**Describe the bug**
After changing the smpt settings to a locale mailserver I tried to test the email connectivity, but no email is sent.
An error is display and logged:
```
2022-08-16T14:26:53+0000 [stdout#info] [D] Key args expected but missing!
2022-08-16T14:26:53+0000 [stdout#info] [D] Received schema dict_keys(['operation']) - Expected dict_keys(['operation', 'args'])
2022-08-16T14:26:53+0000 [twisted.python.log#info] localhost 172.29.0.1 - - [16/Aug/2022:14:26:53 +0000] "PUT /api/admin/config HTTP/1.1" 406 100 23ms - "Mozilla/5.0 (X11; Linux x86_64; rv:103.0) Gecko/20100101 Firefox/103.0"
```

After a while I recognized that exception-emails are successfully sent over the newly configured mail server.

**Screenshots**

- Response: ![grafik](https://user-images.githubusercontent.com/8647114/184905036-2b928c8b-bf4b-4b76-88b6-afc9d1eea785.png)

**Comment by zangels** [comment]
Hello
In my case, the "new submition" or the forced "resend activation link" notification email is sent
But the "new user" notification email is not send

**Comment by evilaliv3** [comment]
Thank you Jose.

1) Which version of the software are you running?
2) you ticket refers to the email test feature but then you asked many
different things. Please if you can be more precise. Are you able to send
the emails?
3) you reported that no email is sent after user creation. This is correct.
The up to date software do not send anymore the credentials to the users
but requires the administrator to send them afterwards

**Comment by zangels** [comment]
Hi @evilaliv3 
Sorry for the mess! You're right!

1) I upgraded to the last version: 4.10.1, and then I configured the our email server.
2) Yes, it's about the email feature. After I configured the server, I hit the test buttom and received the on screen validation error [Missing key atgs].
After that I tested some email notifications (custodian alert; resend user activation link; submit/file a report) these are sent
The only one that wasn't sent is the one that goes when I creat a new user.
3) yes
